### PR TITLE
Improve kubelet logging around container being killed due to hash change

### PR DESCRIPTION
**Pull request by saad-ali**

### CODE REVIEW

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -672,7 +672,7 @@ func (kl *Kubelet) killContainer(dockerContainer *docker.APIContainers) error {
 }
 
 func (kl *Kubelet) killContainerByID(ID, name string) error {
-	glog.V(2).Infof("Killing: %s", ID)
+	glog.V(2).Infof("Killing container with id %s and name %s", ID, name)
```
Comment (thockin) [inline pkg/kubelet/kubelet.go]: We should use %q for these, so that empty strings get caught more obviously.

Comment (saad-ali) [inline pkg/kubelet/kubelet.go]: Good point. Done.

**Comment by saad-ali** [comment]
Updated kubelet.go to use %q instead of %s where it makes sense (container name, pod name, etc.).
Will squash commits after approval.

**Comment by thockin** [comment]
LGTM

**Pull request merged**